requests/utils.py

```diff
@@ -35,7 +35,7 @@
 from .exceptions import (
     InvalidURL, InvalidHeader, FileModeWarning, UnrewindableBodyError)
 
-NETRC_FILES = ('.netrc', '_netrc')
+NETRC_FILES = ('~/.netrc', '~/_netrc')
```

Comment:
I'm a little hesitant to mutate this directly with how long it's been around. Let's keep the formatting inside `get_netrc_auth` if possible.

Reply:
ah makes total sense, wouldn't have thought of that... changing then!

Reply:
done.